### PR TITLE
Add expires property

### PR DESCRIPTION
**Pull request by HelgeKrueger**
See: https://github.com/digitalbazaar/multikey-context/issues/2

### CODE REVIEW

**Comment by HelgeKrueger** [comment]
In https://www.w3.org/ns/credentials/v2 expires is defined as https://w3id.org/security#expiration. This does not match my expectation from https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html#expires. Can somebody clarify?

**Review by dmitrizagidulin — APPROVED** [review]
👍  Thanks @HelgeKrueger!

File: contexts/multikey-v1.jsonld [hunk 1]
```diff
@@ -13,6 +13,10 @@
           "@id": "https://w3id.org/security#controller",
           "@type": "@id"
         },
+        "expires": {
+          "@id": "https://w3id.org/security#expires",
```
Comment (dlongley) [inline contexts/multikey-v1.jsonld]:
```suggestion
          "@id": "https://w3id.org/security#expiration",
```

File: js/context.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ module.exports = {
           '@id': 'https://w3id.org/security#controller',
           '@type': '@id'
         },
+        expires: {
+          '@id': 'https://w3id.org/security#expires',
```
Comment (dlongley) [inline js/context.js]:
```suggestion
          '@id': 'https://w3id.org/security#expiration',
```

**Review by dlongley — CHANGES_REQUESTED** [review]
For historical reasons `...#expiration` has been the full URL used in the security vocabulary. Lots of work was incubated that used that URL mapping and so it's just continued on that way -- we should keep that here to prevent mapping conflicts.

**Comment by dlongley** [comment]
Hmm, I see from the linked comment that some update to the security vocab changed `...#expiration` to `...#expires` -- we'll want to figure out what's going on there. I see that the data integrity context uses `...#expiration` as all of the incubated work has: https://www.w3.org/ns/data-integrity/v1. It's also the case that the VC v1 context uses `...#expiration`: https://www.w3.org/2018/credentials/v1.

File: contexts/multikey-v1.jsonld (hunk 1 above)
Comment (dmitrizagidulin) [reply contexts/multikey-v1.jsonld]: +1 @HelgeKrueger -- the expansion of `key.expires` into `https://w3id.org/security#expiration` may seem odd, but is needed for backwards compatibility.

**Comment by dmitrizagidulin** [comment]
@HelgeKrueger - so I think this PR is almost ready to merge, if you could accept the suggested change from Dave.

**Comment by HelgeKrueger** [comment]
I just approved the requested changes. Does this match the expectation?

**Comment by dmitrizagidulin** [comment]
+1 from me. @dlongley ?

**Review by dlongley — APPROVED** [review]
LGTM.

**Review by msporny — APPROVED** [review]

**Pull request merged**